guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc

```diff
@@ -0,0 +1,62 @@
+[[migrating_pulp_content]]
+
+= Migrating Content to Pulp 3
+
+The time to prepare content for Pulp 3 depends on the amount of content.
+For large systems, this can mean several days of downtime.
+
+Use this procedure to begin migrating content from Pulp 2 to Pulp 3.
+
+.Procedure
+
+. View details of the content you are migrating using the following command:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+# {foreman-maintain} content migration-stats
+----
++
+Use this command as often as necessary during the migration process to determine how long the process will take.
+It also identifies corrupted or missing content that might cause the migration to fail.
+Output is similar to the following:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+Running Retrieve Pulp 2 to Pulp 3 migration statistics
+============================================
+Retrieve Pulp 2 to Pulp 3 migration statistics:
+============Migration Summary================
+Migrated/Total RPMs: 0/367633
+Migrated/Total errata: 0/20780140
+Migrated/Total repositories: 0/33924
+Estimated migration time based on yum content: 47 hours, 23 minutes
+Note: ensure there is sufficient storage space for /var/lib/pulp/published to
+double in size before starting the migration process.
+Check the size of /var/lib/pulp/published with 'du -sh /var/lib/pulp/published/'
+[OK]
+----
++
+. Update the file permissions before the {ProjectServer} upgrade using the following:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+# {foreman-maintain} prep-{TargetVersion}-upgrade
+----
++
+This might take some time on high latency systems.
+. Prepare your content for Pulp migration using the following command:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+# {foreman-maintain} content-prepare
+----
++
+If you know the process will take some time you can use the following command to terminate the process gracefully, whenever necessary, so that you can continue later.
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+# {foreman-maintain} content-prepare abort
+----
++
+. Continue the migration process when convenient using `{foreman-maintain} content-prepare`.
+. The migration process is not fully complete until the upgrade of {ProjectServer} from {ProjectVersionPrevious} to {ProjectVersion} is complete.
```

Comment:
It might be worth mentioning that if you need to start a migration from scratch, you can run:
```
satellite-maintain content migration-reset
```

this would generally only be used for troubleshooting purposes, but probably worth mentioning